hypersync-net-types/src/bloom_filter_wrapper.rs

```diff
@@ -0,0 +1,97 @@
+use hex::{decode, encode};
+use std::fmt;
+
+use serde::{
+    de::{self, Visitor},
+    Deserialize, Deserializer, Serialize, Serializer,
+};
+
+#[derive(Debug, Clone)]
+pub struct FilterWrapper(sbbf_rs_safe::Filter);
+
+impl PartialEq for FilterWrapper {
+    fn eq(&self, other: &Self) -> bool {
+        self.0.as_bytes() == other.0.as_bytes()
```

Comment:
Not sure if this is okay

Reply:
should be ok but we probably don't need it

Reply:
Just had to include it because removing `PartialEq` from Query breaks a bunch of things